Tests/SentryTests/SentryOptionsTest.m

```diff
@@ -48,14 +48,27 @@ - (void)testInvalidDsn {
     XCTAssertEqual(kSentryErrorInvalidDsnError, error.code);
     XCTAssertNil(options);
 }
-    
+
 - (void)testRelease {
     SentryOptions *options = [self getValidOptions:@{}];
     XCTAssertNil(options.releaseName);
-    
+
     options = [self getValidOptions:@{@"release": @"abc"}];
     XCTAssertEqualObjects(options.releaseName, @"abc");
 }
+
+- (void)testNoReleaseSetUsesDefault {
```

Comment:
The tests `testRelease` and `testNoReleaseSetUsesDefault` are failing for me. If we have one for no release set and another one for setting the release, I'm good.

Reply:
tests are there, lets see if CI is happy now